indico_hub/server.py

```diff
@@ -88,16 +88,16 @@ def update_instance(uuid, **kwargs):
     inst = Instance.query.filter_by(uuid=uuid).first()
     if inst is None:
         abort(404, description='BAD REQUEST: user already exist')
-    if(kwargs['enabled'] == False):
-        #unregister instance
+    if kwargs['enabled'] is False:
+        # unregister instance
         db.session.delete(inst)
         db.session.commit()
-        return InstanceSchema().dumps(inst), 201
+        return InstanceSchema.dumps(inst), 201
 
     for attr in kwargs:
         setattr(inst, attr, kwargs[attr])
     db.session.commit()
-    return InstanceSchema().dumps(inst), 200
+    return (InstanceSchema().dumps(inst)), 200
```

Comment:
why those extra parentheses?

Reply:
used to be jsonified, but I edited anyways
Solved!